backend/functions/.funcignore

```diff
@@ -13,3 +13,4 @@ README.md
 .env
 .eslint*
 *.test.*
+node_modules/azure-functions-core-tools
```

Comment:
Wondering if we need to include anything else here? Maybe even the complete node_modules

Reply:
Let's revisit this another time.